ansible_ai_connect/main/tests/test_views.py

```diff
@@ -185,3 +187,23 @@ def test_get_view(self):
         r = self.client.get(reverse("me_summary"))
         self.assertEqual(r.status_code, 200)
         self.assertContains(r, "Logged in as: test_user_name")
+
+    @override_settings(ANSIBLE_AI_ENABLE_TECH_PREVIEW=False)
+    def test_get_view_trial(self):
+        user = create_user_with_provider(provider=USER_SOCIAL_AUTH_PROVIDER_OIDC)
+        self.client.force_login(user=user)
+
+        trial_plan, _ = Plan.objects.get_or_create(name="trial of 90 days", expires_after="90 days")
+        user.plans.add(trial_plan)
+
+        r = self.client.get(reverse("me_summary"))
+        self.assertEqual(r.status_code, 200)
+        content = r.json()["content"]
+        expectation = """
+        Logged in as: test_user_name
+
+        User Type: Trial
+        Plan: trial of 90 days
+        Expiration: 2024-10-14
```

Comment:
This is likely to fail tomorrow or next week or next month?

Reply:
ahah, this is embarrassingly true :-).